docker/pypi/wmagent/bin/manage-common.sh

```diff
@@ -219,6 +219,31 @@ _sql_db_isclean(){
     esac
 }
 
+_couch_db_isclean(){
+  # assume that we can connect to local CouchDB
+  # assume that env contains local couch user and password.
+  echo "$FUNCNAME: Checking if local couch db is empty"
+  agent_summary=$(curl -s -u "$COUCH_USER:$COUCH_PASS" http://$COUCH_HOST:$COUCH_PORT/wmagent_summary)
+
+  if ! type jq > /dev/null ; then
+    echo "$FUNCNAME: jq is not installed, we can not check if couchdb is empty"
+    return $(false)
+  fi
+  agent_summary_notclean=$(jq "select(.doc_count > 1 or .doc_del_count > 0)" <<< $agent_summary)
+  
+  if [ -n "${agent_summary_notclean}" ]; then
+    # the variable is not empty -> means that couch is not empty
+    echo "$FUNCNAME: local couch is not empty. we can not init wmagent"
+    doc_count=$(jq ".doc_count" <<< $agent_summary)
+    doc_del_count=$(jq ".doc_del_count" <<< $agent_summary)
+    echo "$FUNCNAME: it contains $doc_count docs, $doc_del_count deleted docs"
+    return $(false)
+  else
+    # the variable is empty -> means that couch is empty
+    echo "$FUNCNAME: local couch db is empty. we can init wmagent"
```

Comment:
"We cannot init WMAgent"?

Reply:
I mean, I agree that i can use WMAgent, but still if the local CouchDB is empty then we can proceed and init the agent